app/stylesheets/main.scss

```diff
@@ -38,6 +48,9 @@ body, html {
   display: flex;
   flex-direction: row;
   overflow: hidden; // required for footer bar to display in Firefox
+  textarea {
+    line-height: 1.5em;
```

Comment:
This PR should only address syntactical changes, and not functionality or appearance. This should be left to a separate PR.

Reply:
This is to fix issue https://github.com/sn-extensions/markdown-basic/issues/3. It's an easy fix. Are you sure you want to remove it?